core/ui/public/i18n/language.json

```diff
@@ -1023,5 +1025,29 @@
     "documentation": "Documentation",
     "helpdesk": "Helpdesk",
     "animated_icons_by": "Animated icons by"
+  },
+  "smarthost": {
+    "title":"Smarthost settings",
+    "enabled":"Enable the smarthost provider",
+    "hostname_label":"Hostname",
+    "hostname_label_tooltip":"Write the fully qualified domain name where to use the SMTP server",
+    "hostname_placeholder":"smtp.domain.com",
+    "username_label":"Username",
+    "username_label_tooltip":"Write the user account or let it blanck if the server requires no authentication",
+    "username_placeholder":"foo@domain.com",
+    "password_label":"Password",
+    "port_label":"Port",
+    "port_label_tooltip":"Write the TCP port where to reach the SMTP server",
+    "port_placeholder":"587",
+    "tls":"Enable STARTTLS encryption",
+    "tls_verify":"Verify the TLS certificate",
+    "connection_timeout_error":"The connection issued a timeout error, verify the port and the fully qualified domain name",
+    "cannot_authenticate_to_server":"We cannot authenticate, it is probably due to a bad login or bad password",
+    "cannot_connect_to_server":" We cannot connect to the SMTP Server, please try again.",
```

Comment:
```suggestion
    "cannot_connect_to_server":" Cannot connect to the SMTP Server, please try again.",
```

Reply:
Same as above, we never use "we" in UI messages, I would stick with a more impersonal form